components/widgets/land-use/forestry-employment/index.js

```diff
@@ -12,28 +12,29 @@ export default {
     {
       key: 'year',
       label: 'year',
-      options: [1990, 2000, 2005, 2010].map(y => ({ label: y, value: y })),
-      type: 'select'
-    }
+      options: [1990, 2000, 2005, 2010].map((y) => ({ label: y, value: y })),
+      type: 'select',
+    },
   ],
   chartType: 'pieChart',
   dataType: 'fao',
   metaKey: 'widget_forestry_employment',
   sortOrder: {
-    landUse: 3
+    landUse: 3,
   },
   settings: {
-    year: 2010
+    year: 2010,
+    unit: 'countsK',
```

Comment:
Note: This is the only relevant change in this file. The other changes were due VS Code auto-formatting.